lib/carbon/util.py

```diff
@@ -100,15 +100,23 @@ def parseDestinations(destination_strings):
   destinations = []
 
   for dest_string in destination_strings:
-    parts = dest_string.strip().split(':')
-    if len(parts) == 2:
-      server, port = parts
-      instance = None
-    elif len(parts) == 3:
-      server, port, instance = parts
+    s = dest_string.strip()
```

Comment:
Could this be moved into a specific function ? (to make it easier to use ipaddress module later)